index.js

```diff
@@ -0,0 +1,6 @@
+import { } from './modules/TaskLine.js';
+import { } from './modules/AddTask.js';
+import { } from './modules/listInput.js';
+import { } from './modules/localstorage.js';
+import { } from './modules/RemoveTask.js';
+import { } from './modules/UpdateTask.js';
```

Comment:
It's really interesting what you've done with your index.js keep up the great work 👍

Reply:
Thank you :)